data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/worker/client/X509TrustAllManager.java

```diff
@@ -0,0 +1,22 @@
+package org.opensearch.dataprepper.plugins.source.opensearch.worker.client;
+
+import javax.net.ssl.X509TrustManager;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+public class X509TrustAllManager implements X509TrustManager {
```

Comment:
This is a direct copy from the opensearch sink (https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/X509TrustAllManager.java)